### DISCUSSION TITLE
zh/guide/drivers/s3

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#65] [Announcements]
# zh/guide/drivers/s3

AList v3的文档

https://alist.nn.ci/zh/guide/drivers/s3.html

**Comment by wuwuhen** [comment]
移动云对象存储挂载不了，对象存储 EOS（Elastic Object Storage），是还不支持吗？

**Reply by anwen-anyi** [reply]
可以挂啊~移动云还要预存100 才能用........

**Comment by jgc9527** [comment]
能够支持自搭建minio的对象储存吗

**Comment by ssslos** [comment]
oss提示：
failed get link: InvalidParameter: 1 validation error(s) found. - minimum field size of 1, GetObjectInput.Key.

**Comment by huangdiyou** [comment]
Failed create storage in database: UNIQUE constraint failed: x_storages.mount_path

**Comment by adonis-lau** [comment]
请问Oracle云的储存桶有办法挂载吗？我尝试了一下，拼接后的地址不对。

**Reply by anwen-anyi** [reply]
Oracle云的储存桶应该也是AWS S3协议的，但是我没`Oracle`的账号无法测试

**Reply by hcllmsx** [reply]
我有甲骨文云账号，可以给你测试，我想添加存储桶，我怎么联系你

**Reply by anwen-anyi** [reply]
> 我有甲骨文云账号，可以给你测试，我想添加存储桶，我怎么联系你

在tg群 找安稳

**Comment by chenpeng198877** [comment]
b2 无法获取到正确上传时间和文件大小！所有内容都时最后一个上传资源的时间，希望能尽快修复一下，谢谢！

**Reply by hdsfgsgrlsgtw12** [reply]
请问这个修复了吗？我文件夹的时间都是同一时间，文件时间还是实时更新的
<img width="356" alt="image" src="https://github.com/alist-org/docs/assets/45323722/561d7fb4-9150-44fc-80cb-f306f5f08171">

**Comment by gisxlab** [comment]
请问有办法把这类开放数据挂过来吗？
https://registry.opendata.aws/
譬如https://registry.opendata.aws/copernicus-dem/

**Comment by xiwangly2** [comment]
目前我挂载AWS S3上传大文件（大于4GB的文件）显示上传成功，但是实际没有上传成功，这个问题的发生频率是偶尔，我已经排除了我的机器的网络和硬件配额的问题

**Comment by yeyudoc** [comment]
rclone 提供的s3该怎么添加，我尝试后一直失败

**Comment by Zac-Linley** [comment]
添加minio时，配置Endpoint

```
Endpoint：http://127.0.0.1:8888
```

会报错：

```
failed get objs: failed to list objs: AuthorizationHeaderMalformed: The authorization header is malformed; the region is wrong; expecting 'localhost'. status code: 400, request id: 179BD3EC77F24640, host id: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
```

**Reply by leiax00** [reply]
我添加minio后, 图片无法预览; 预览url拿外边访问能够正常下载文件

**Comment by leiax00** [comment]
我添加minio后, 图片无法预览; 预览url拿外边访问能够正常下载文件

**Reply by leiax00** [reply]
哦, 我开启web代理后就可以了

**Comment by xiaolan66** [comment]
{对象存储}默认上传是什么方式呢，文件先到alist服务器然后再到oss，还是直接到oss不用经过中转？

**Reply by anwen-anyi** [reply]
> {对象存储}默认上传是什么方式呢，文件先到alist服务器然后再到oss，还是直接到oss不用经过中转？

所有的都是先上传到alist再由alist上传到终端

**Reply by xiaolan66** [reply]
这样有个比较严重的弊端，我服务器速度慢的话，就拖累了oss

**Comment by Sakana-1314** [comment]
cf R2 的预览一直是403，有佬教一下吗？

**Reply by iislong** [reply]
Custom Host格式错了吧，网址后面不要/

**Reply by Sakana-1314** [reply]
> Custom Host格式错了吧，网址后面不要/

不是挂载失败。能右键下载，但是是预览403。应该是跨域策略设置得不对
